contracts/utils/Create3Address.sol

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+/**
+ * @title Create3Address contract
+ * @notice This contract can be used to predict the deterministic deployment address of a contract deployed with the `CREATE3` technique.
+ */
+contract Create3Address {
+    /// @dev bytecode hash of the CreateDeploy helper contract
+    bytes32 internal immutable createDeployBytecodeHash;
```

Comment:
Since we're fixing this value, this doesn't need to be an immutable. Maybe fix the bytecode hash? You can also inherit/import the constant bytecode and it's hash from a separate contract file